freqtrade/rpc/rpc.py

```diff
@@ -135,12 +134,11 @@ def _rpc_status_table(self) -> DataFrame:
                     trade.pair,
                     shorten_date(arrow.get(trade.open_date).humanize(only_distance=True)),
                     f'{trade_perc:.2f}%'
+
                 ])
 
             columns = ['ID', 'Pair', 'Since', 'Profit']
-            df_statuses = DataFrame.from_records(trades_list, columns=columns)
-            df_statuses = df_statuses.set_index(columns[0])
-            return df_statuses
+            return trades_list, columns
```

Comment:
why have you changed it from returning the dataframe to this tuple? Due to performance or something else?

Reply:
if you look at where this is used - then converting this to a DataFrame makes no sense.

The output of this function is then passed to tabulate - with "columns" as column headers ...
the same can be accomplished by keeping this as list - and passing the headers back - removing the overhead of creating (and uncreating) a Dataframe.

I did not measure the performance - but just thinking about it - it's avoiding a unnecessary conversion to dataframes.